src/main/java/appeng/util/inv/IInternalItemHandler.java

```diff
@@ -20,10 +20,12 @@
 
 
 import net.minecraft.item.ItemStack;
+import net.minecraftforge.items.IItemHandlerModifiable;
 
 
-public interface IInventoryWrapper
+public interface IInternalItemHandler extends IItemHandlerModifiable
 {
+	public boolean isItemValidForSlot( int slot, ItemStack stack );
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Redundant 'public' modifier. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.modifier.RedundantModifierCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Avoid modifiers which are implied by the context [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=pmd%3AUnusedModifier)